internal/services/devcenter/dev_center_dev_box_definition_resource_test.go

```diff
@@ -162,11 +165,12 @@ provider "azurerm" {
 }
 
 resource "azurerm_dev_center_dev_box_definition" "test" {
-  name               = "acctest-dcet-%d"
-  location           = azurerm_resource_group.test.location
-  dev_center_id      = azurerm_dev_center.test.id
-  image_reference_id = "${azurerm_dev_center.test.id}/galleries/default/images/microsoftvisualstudio_visualstudioplustools_vs-2022-ent-general-win11-m365-gen2"
-  sku_name           = "general_i_8c32gb512ssd_v2"
+  name                      = "acctest-dcet-%d"
+  location                  = azurerm_resource_group.test.location
+  dev_center_id             = azurerm_dev_center.test.id
+  image_reference_id        = "${azurerm_dev_center.test.id}/galleries/default/images/microsoftwindowsdesktop_windows-ent-cpc_win11-24h2-ent-cpc"
+  hibernate_support_enabled = true
```

Comment:
Since this config is used to test updates to properties, we should ensure this has a different value than the `complete` test

```suggestion
  hibernate_support_enabled = false
```

Reply:
Got it.